Week3/Company_create_insert_fk.sql

```diff
@@ -0,0 +1,104 @@
+CREATE DATABASE IF NOT EXISTS Company;
+USE Company;
+
+CREATE TABLE IF NOT EXISTS Employee (
+  eID INT(4) NOT NULL,
+  eName VARCHAR(45) NOT NULL,
+  initials VARCHAR(10) NULL DEFAULT NULL,
+  dateOfBirth DATE NOT NULL,
+  position VARCHAR(20) NOT NULL,
+  salary DECIMAL(7,2) NOT NULL,
+  room VARCHAR(10) NOT NULL,
+  bossNr INT(4) NULL DEFAULT NULL,
+  dID INT(2) NOT NULL,
```

Comment:
You're not paying by the letter, you can feel free to use 'employeeID' and 'departmentID' instead of 'eID' and 'dID'. :stuck_out_tongue: 

And you should, because they're more clear and more readable.